interface/controller/student_controller.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"encoding/csv"
+	"errors"
+	"golang-bootcamp-2020/domain/model"
+	"io"
+	"log"
+	"os"
+	"strconv"
+)
+
+type StudentController interface {
+	GetStudents() []model.Student
+}
+
+func GetStudents() []model.Student {
+	csvFile, err := os.Open("infrastructure/datastore/dataFile.csv")
+	check(err)
+
+	var students []model.Student = nil
```

Comment:
Why initialize it as `nil`?
```go 
students := []model.Student{}
```

Reply:
just initialize as in other languages, it won't happen again in go